### PR TITLE
Use the local test web server for all tests

### PR DESCRIPTION
**Pull request by theseion**
Fixed issues with false positive test results

### CODE REVIEW

**Comment by theseion** [comment]
Note: not all tests will succeed. Originally, some tests succeeded even though they shouldn't have. After fixing that, they now fail. That will be cleaned up with introduction of log marking.

File: runner/run.go [hunk 1]
```diff
@@ -35,6 +36,7 @@ func Run(include string, exclude string, showTime bool, output bool, ftwtests []
 			// if we received a particular testid, skip until we find it
 			if needToSkipTest(include, exclude, t.TestTitle, tests.Meta.Enabled) {
 				addResultToStats(Skipped, t.TestTitle, &stats)
+				log.Info().Msgf("Skipping test %s", t.TestTitle)
```
Comment (fzipi) [inline runner/run.go:39]: The problem here is that we cannot control if this is logged in quiet mode.

**Review by fzipi — COMMENTED** [review]
Other than that, looks nice!

**Comment by fzipi** [comment]
So we should merge this one despite tests failing?

**Comment by theseion** [comment]
Depends. The other PR is based on this one, so it includes these changes too. I just wanted to separate the change. Tests in the other PR are all green.

File: runner/run.go (hunk 1 above)
Comment (theseion) [reply runner/run.go]: I've changed it to respect quiet mode. Better?

**Pull request merged**